src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/DemoExample.razor.scss

```
@@ -1,4 +1,4 @@
-﻿@import '../Styles/abstracts/_functions.scss';
+@import '../Styles/abstracts/_functions.scss';
```

Stylelint (17.5.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)